packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -189,19 +170,20 @@ export const LongLabelsWithWrap: ComponentStory<typeof FormFieldNext> = (
 ) => {
   return (
     <FlowLayout style={{ width: "366px" }}>
-      <FormFieldNext
-        label="Form Field label that's extra long. Showing that labels wrap around to the next line."
-        {...props}
-      >
+      <FormFieldNext {...props}>
+        <Label>
+          Form Field label that's extra long. Showing that labels wrap around to
+          the next line.
+        </Label>
         <InputNext defaultValue="Primary Input value" />
       </FormFieldNext>
-      <FormFieldNext
-        readOnly
-        label="Form Field label"
-        helperText="Helper text that's very long. Additional text explaining that this is a readonly Form Field."
-        {...props}
-      >
+      <FormFieldNext readOnly {...props}>
```

Comment:
Do we need to add this readonly example here? I'd replace it with a left aligned label, or even move into a LabelPlacement story